src/helpers/dataframe/sort.ts

```diff
@@ -8,6 +8,7 @@ export function sortableDataFrame<M extends Obj, C extends Obj>(
 ): DataFrame<M, C> {
   // If sortableColumns is not provided, make all columns sortable.
   const sortableColumns = options?.sortableColumns ?? new Set(data.columnDescriptors.map(c => c.name))
+  const exclusiveSort = options?.exclusiveSort ?? data.exclusiveSort
```

Comment:
I think we can remove options?.exclusiveSort, the only way to pass it should be through the DataFrame, no?